packages/angular_devkit/core/node/resolve.ts

```diff
@@ -259,7 +259,7 @@ export function resolve(x: string, options: ResolveOptions): string {
   }
 
   function nodeModulesPaths(start: string, opts: ResolveOptions) {
-    const modules = ['node_modules'];
+    const modules = ['node_modules', path.join('npm', 'node_modules')];
```

Comment:
Where does the magic string `npm` come from here? This looks like it should be documented and maybe configurable, since `@angular-devkit/core` is used outside of this repository.

Reply:
This is used at runtime during an ng build or serve.
Bazel should have no effect here.

Reply:
I reverted this change, and instead added a "fake" node_modules directory so that the resolve() test works.